tracks/java/exercises/bird-watcher/mentoring.md

```diff
@@ -0,0 +1,108 @@
+# Mentoring
+
+## Reasonable solutions
+
+### Using for loop iteration
+
+```java
+class BirdWatcher {
+    private int[] birdsPerDay;
+
+    public BirdWatcher(int[] birdsPerDay) {
+        this.birdsPerDay = birdsPerDay;
+    }
+
+    public int[] getLastWeek() {
+        return birdsPerDay;
+    }
+
+    public int getToday() {
+        return birdsPerDay.length > 0 ? birdsPerDay[birdsPerDay.length - 1] : 0;
+    }
+
+    public int incrementTodaysCount() {
+        return ++birdsPerDay[birdsPerDay.length - 1];
+    }
+
+    public boolean hasDayWithoutBirds() {
+        for (int dailyBirdCount : birdsPerDay) {
+            if (dailyBirdCount == 0) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public int getCountForFirstDays(int numberOfDays) {
+        int visitingBirdsCount = 0;
+        for (int i = 0; i < Math.min(numberOfDays, birdsPerDay.length); i++) {
+            visitingBirdsCount += birdsPerDay[i];
+        }
+        return visitingBirdsCount;
+    }
+
+    public int getBusyDays() {
+        int busyDaysCount = 0;
+        for (int dailyBirdCount : birdsPerDay) {
+            if (dailyBirdCount >= 5) {
+                busyDaysCount++;
+            }
+        }
+        return busyDaysCount;
+    }
+}
+```
```

Comment:
This is a bit different than the exemplar solution https://github.com/exercism/java/blob/main/exercises/concept/bird-watcher/.meta/src/reference/java/BirdWatcher.java

For example the ternary operators aren't required to learn about arrays so you shouldn't tell students to use them. They might not know about them yet.

Reply:
As my other comment said, I will just remove the Reasonable Solutions altogether since the exemplar is now part of the mentoring view.